config/locales/en/casework.yml

```diff
@@ -125,6 +145,12 @@ en:
     name: Name
     national_crime_team: CAT 2
     national_insurance_number: National Insurance number
+    national_savings_certificates: National Savings Certificates
```

Comment:
@timpeat What do you think of scoping/namespacing capital components?

Reply:
@ripan I'm still undecided. I thought perhaps lets decide when we've tied down what to do with the locals on Apply and copy that pattern.